scripts/src/internal/services.ts

```diff
@@ -101,7 +102,12 @@ async function getPaymentJWT(order: db.Order, appId: string, user: User): Promis
 }
 
 export async function paymentComplete(payment: CompletedPayment) {
-	const order = await db.Order.getOne({ orderId: payment.id });
+	// if I can find the "paytment.id" in the redis, it means its a memo and the value is the real order id
+	const redis = getRedisClient();
+	const incomingOrderId = await redis.async.get("transfer_" + payment.id);
```

Comment:
please use a prefix with colon and use string substitution: 
```
.get(`transfer:${payment.id}`)
```

Reply:
done